pycode/memilio-surrogatemodel/memilio/surrogatemodel/ode_secir_groups/network_architectures.py

```diff
@@ -0,0 +1,102 @@
+#############################################################################
+# Copyright (C) 2020-2023 German Aerospace Center (DLR-SC)
+#
+# Authors: Agatha Schmidt, Henrik Zunker
+#
+# Contact: Martin J. Kuehn <Martin.Kuehn@DLR.de>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#############################################################################
+import tensorflow as tf
+
+
+def mlp_multi_input_single_output(num_age_groups=6):
+    """! Simple MLP Network which takes the compartments for multiple time steps as input and
+    returns the 8 compartments for all six age groups for one single time step.
+
+    Reshaping adds an extra dimension to the output, so the shape of the output is 1x48.
+    This makes the shape comparable to that of the multi-output models.
+
+    @param num_age_groups Number of age groups in population.
+    """
+    model = tf.keras.Sequential([
+        tf.keras.layers.Flatten(),
+        tf.keras.layers.Dense(units=32, activation='relu'),
+        tf.keras.layers.Dense(units=32, activation='relu'),
+        tf.keras.layers.Dense(units=8*num_age_groups),
+        tf.keras.layers.Reshape([1, -1]), ])
+    return model
+
+
+def mlp_multi_input_multi_output(label_width, num_age_groups=6):
+    """! Simple MLP Network which takes the compartments for multiple time steps as input and
+    returns the 8 compartments for all age groups for multiple time steps in the future. 
+
+    Reshaping adds an extra dimension to the output, so the shape of the output is 30x48.
+    This makes the shape comparable to that of the multi-output models.
```

Comment:
Isn't the shape num_time_stepsx48 so only 30x48 if you want to predict 30 timesteps?